compiam/melody/raga_recognition/deepsrgm/__init__.py

```diff
@@ -182,24 +188,33 @@ def get_features(
 
         else:
             try:
+                import essentia.standard as estd
                 melodia = compiam.melody.pitch_extraction.Melodia
-                melodia = melodia()
+                melodia = melodia(sampleRate=self.sample_rate)
                 tonic_extraction = compiam.melody.tonic_identification.TonicIndianMultiPitch
-                tonic_extraction = tonic_extraction()
+                tonic_extraction = tonic_extraction(sampleRate=self.sample_rate)
             except:
                 raise ImportError(
                     "In order to use these tools to extract the features you need to have essentia installed."
                     "Please install essentia using: pip install essentia"
                 )
-            if not os.path.exists(audio_path):
-                raise FileNotFoundError("Input audio not found.")
+
+            if isinstance(input_data, str):
+                if not os.path.exists(input_data):
+                    raise FileNotFoundError("Target audio not found.")
+                audio = estd.MonoLoader(filename=input_data, sampleRate=self.sample_rate)()
+            elif isinstance(input_data, np.ndarray):
+                print("Resampling... (input sampling rate is {}Hz, make sure this is correct)".format(input_sr))
```

Comment:
fstrings are a bit easier to read

Reply:
not that important.... just using something like `"{input_str}Hz, make sure..."`

Reply:
Ok! So as I am saying above, maybe we could, again, move all `prints` to `logger`, in a PR for that, since we have a lot of those around and using `Print` is probably not the best practise.